cv2_enumerate_cameras/linux_backend.py

```diff
@@ -25,6 +25,8 @@ def read_line(*args):
 def cameras_generator(apiPreference):
     for path in glob.glob('/dev/video*'):
         device_name = os.path.basename(path)
+        if not device_name[5:].isdigit():
+            break
```

Comment:
OpenCV only supports purely numeric video device names (i.e. /dev/video1). Here it should continue instead of break.

Reference: https://github.com/opencv/opencv/blob/6cc166985d9e30d78a5a117e54dec68ae0b477ca/modules/videoio/src/cap_v4l.cpp#L993

Reply:
> OpenCV only supports purely numeric video device names (i.e. /dev/video1). Here it should continue instead of break.
> 
> Reference: https://github.com/opencv/opencv/blob/6cc166985d9e30d78a5a117e54dec68ae0b477ca/modules/videoio/src/cap_v4l.cpp#L993

You are right, it was my mistake. Look at my last commit, I already fixed that and some other things